server/tests/test_inference.py

```
@@ -0,0 +1,67 @@
+# Add the project root to sys.path
+sys.path.append(str(Path(__file__).parent.parent.parent))
+
+from server.backend.mlx_runner import MLXRunner
+from server.cache_utils import get_model_path
+
+def test_inference():
+    model_spec = "driaforall/mem-agent-mlx-4bit"
+    print(f"Testing inference with model: {model_spec}")
+    
+    try:
+        model_path, model_name, commit_hash = get_model_path(model_spec)
```

Ruff (0.14.11)

Unpacked variable `model_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)